src/parser.ts

```diff
@@ -9095,10 +9119,6 @@ export function parseJSXClosingFragment(
 export function parseJSXChildren(parser: ParserState, context: Context): ESTree.JSXChild[] {
   const children: ESTree.JSXChild[] = [];
   while (parser.getToken() !== Token.JSXClose) {
-    parser.index = parser.tokenPos = parser.startPos;
-    parser.column = parser.colPos = parser.startColumn;
-    parser.line = parser.linePos = parser.startLine;
-    scanJSXToken(parser, context);
```

Comment:
This reset index to startPos can scan last token again. It look weird, and removing it didn't hurt any test!
Maybe a left over junk.